agent/script/parallel_script.go

```diff
@@ -66,6 +66,23 @@ func (s ParallelScript) Run() error {
 	return s.summarizeErrs(passedScripts, failedScripts)
 }
 
+func (s ParallelScript) Cancel() error {
+	s.logger.Debug(s.logTag, "Canceling a parallel script")
+	existingScripts := s.findExistingScripts(s.allScripts)
+	for _, script := range existingScripts {
+		if script, ok := script.(CancellableScript); ok {
```

Comment:
should we just add Cancel to the Script interface. For non drain scripts we can implement it as always returning an error for now?